src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -18,12 +18,12 @@
             [schema.core :as schema]
             [cheshire.core :as cheshire]
             [liberator.core :refer [defresource]]
-            ;[liberator.dev :as liberator-dev]
```

Comment:
?

Reply:
My IDEA had done something weird with the indentation here, so I noticed it, and thought it was leftover, and deleted it. If you think it's in here for a good reason I'll put it back.